CMakeLists.txt

```diff
@@ -60,74 +60,92 @@ if( ENABLE_TESTING )
     find_package( test-drive REQUIRED )
 endif()
 
+# Optionally use PackageCompiler.jl to build standalone libtrixi.so
+option( USE_PACKAGE_COMPILER "Build standalone libtrixi.so using PackageCompiler.jl" )
+if( USE_PACKAGE_COMPILER )
+    if ( NOT DEFINED JULIA_PROJECT_PATH )
+        message( FATAL_ERROR "JULIA_PROJECT_PATH needs to be set for PackageCompiler.jl.")
+    endif()
 
+    # no sources
+    add_library( ${PROJECT_NAME} INTERFACE build-pc/lib/libtrixi.so )
+    add_custom_command( OUTPUT build-pc/lib/libtrixi.so
+                        COMMENT "Building ${PROJECT_NAME} with PackageCompiler.jl..."
+                        COMMAND ${JULIA_EXECUTABLE}
+                                --project=${CMAKE_SOURCE_DIR}/LibTrixi.jl/lib
+                                ${CMAKE_SOURCE_DIR}/LibTrixi.jl/lib/build.jl
+                                ${JULIA_PROJECT_PATH}
```

Comment:
I think `JULIA_PROJECT_PATH` is not an optimal name. It *hints* at being something used by Julia, but it is not (we use it in libtrixi). What about renaming it (here and elsewhere) to `LIBTRIXI_JULIA_PROJECT`? This is similar enough to Julia's own [`JULIA_PROJECT`](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_PROJECT) such that those familiar with Julia will know what this is about, while clearly indicating that this is not used by Julia directly but just by us?